.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,51 @@
+name: Lint and Test Charts
+
+on:
+  push:
+    branches-ignore:
+    - 'main'
+    - 'repository'
+    
+  pull_request:
+    branches:
+    - '**'
```

Comment:
I'm not sure what the workflow on this repository is, but if `main` is protected from pushes and PRs are required then this can be just
```
on: pull_request
```

Reply:
I wasn't sure of the workflow, either. The intention is to lint and test everywhere except the `main` and `gh-pages` (formerly `repository`) branches. 

I'll assume `on: pull_request` is correct as per your suggestion.